schemas/io.snowplow.foundation/content/jsonschema/1-0-0

```diff
@@ -0,0 +1,37 @@
+{
+  "description": "Schema for a content entity",
+  "properties": {
+    "name": {
+      "type": "string",
+      "maxLength": 255
+    },
+    "id": {
+      "type": "string",
+      "maxLength": 255
+    },
+    "category": {
+      "type": "string",
+      "maxLength": 255
+    },
+    "date_published": {
+      "type": "string",
+      "maxLength": 255
+    },
+    "author": {
+      "type": "string",
+      "maxLength": 255
+    }
+  },
+  "additionalProperties": false,
+  "type": "object",
+  "required": [
+    "name"
+  ],
+  "self": {
+    "vendor": "io.snowplow.foundation",
```

Comment:
What does `foundation` represent/mean? Are these not the recipe schemas from Try Snowplow? Why not `io.snowplow.recipe` or a final part that represents the "usecase" we typically see this schema used in `io.snowplow.content`?

Reply:
I was OK with the name "foundation" for the vendor, meaning like a collection of schemas that can be used when first starting off with Snowplow, before graduating to more advanced use cases when the user would want to design their own bespoke schemas.

But... I find the names of the schemas a bit strange.  The names "content" and "conversion" don't mean anything to me, but maybe that's because I don't come from the world of web analytics.

I don't have any problem with the schemas though, so I'd be OK to approve this release.

Reply:
`foundation` was the group consensus for what Ian describes

we didn't want to go with use-case specific names in case we later want to add more advanced content / conversion / funnel schemas that would then clash with these

Reply:
I'd be happy with `recipe` too, I don't who that decision is ultimately up to

Reply:
Sorry to be that guy...

Now that I understand what `foundation` means, as in foundational... I think I prefer it over receipes as its more generic and allows these schemas to be used in a more broader context than just the recipe context.